specs/latest/2.0/index.html

```diff
@@ -697,13 +697,13 @@ <h2 class="no-toc">Table of contents</h2>
   /* Buffer objects */
   // WebGL1:
   void bufferData(GLenum target, GLsizeiptr size, GLenum usage);
-  void bufferData(GLenum target, BufferDataSource? srcData, GLenum usage);
-  void bufferSubData(GLenum target, GLintptr dstByteOffset, BufferDataSource? srcData);
+  void bufferData(GLenum target, ArrayBuffer srcData, GLenum usage);
```

Comment:
Since the current intent is to match the nullability of the WebGL 1.0 entry points, this should be "ArrayBuffer?".